### ISSUE TITLE
Contribitor loading using Sobol API

### ISSUE THREAD

**Issue by 0xbaer** [Muziris-Labs/tl-bank#2]

**Comment by kbuger** [comment]
You can get almost any information that is on Sobol by the API. 

https://replit.com/join/ksbqezmphp-raybanklesseth
I have made this discord bot that communicates with sobol api and
1- Assign Role : Assigns a role on Sobol according to Otterspace Badge held
2- Get wallet :  Retrieves the wallet information of contributors of a role
3- Message Team : Mentions people on discord, who have a role on sobol but not on discord.

Basically you can get all contributor, with their teams and roles. that is given. We can get a wallet address if they shared it with BanklessDAO Sobol Instance.
